src/ucp/api/ucp_def.h

```diff
@@ -386,12 +386,20 @@ typedef void (*ucp_tag_recv_callback_t)(void *request, ucs_status_t status,
  *
  * The enumeration allows specifying which events are expected on wakeup, though
  * empty events are possible.
+ *
+ * @note UCP doesn't generate POLLOUT like events (see poll man pages) since
+ * it's always ready to initiate outgoing operations (e.g any type of sending,
+ * atomic or RMA). Hoverer, they can be pended due to limitation of resources.
```

Comment:
s/pended/queued/
need to note that send completions are reported by POLLIN-like events